packages/drafts-realm/event.gts

```diff
@@ -0,0 +1,251 @@
+import { Person as PersonCard } from './person';
+import StringCard from 'https://cardstack.com/base/string';
+import { array, fn } from '@ember/helper';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+import {
+  Component,
+  field,
+  contains,
+  CardDef,
+  linksTo,
+} from 'https://cardstack.com/base/card-api';
+import TextAreaCard from '../base/text-area';
+import BooleanField from '../base/boolean';
+import DateField from '../base/date';
+import TimeCard from '../base/time';
+import {
+  BoxelDropdown,
+  FieldContainer,
+  Menu,
+} from '@cardstack/boxel-ui/components';
+
+import { menuItem } from '@cardstack/boxel-ui/helpers';
+
+class Edit extends Component<typeof Event> {
+  @tracked eventType = this.args.model.eventType;
+
+  @action updateEventType(type: string) {
+    this.eventType = type;
+    this.args.model.eventType = type;
+  }
+
+  <template>
+    <div class='row'>
+      <FieldContainer @label='Subject' @tag='label' class='field column'>
+        <@fields.subject />
+      </FieldContainer>
+      <FieldContainer @label='Assigned to' @tag='label' class='field column'>
+        <@fields.assignee />
+      </FieldContainer>
+    </div>
+    <div class='row'>
+      <FieldContainer @label='Location' @tag='label' class='field column'>
+        <@fields.location />
+      </FieldContainer>
+      <div class='column' />
+    </div>
+    <div class='row'>
+      <FieldContainer @label='Start' @tag='label' class='field column'>
+        <div class='inner-row'>
+          <@fields.startDate />
+          <div class='divider' />
+          <@fields.startTime />
+        </div>
+      </FieldContainer>
+      <div class='column' />
+    </div>
+    <div class='row'>
+      <FieldContainer @label='End' @tag='label' class='field column'>
+        <div class='inner-row'>
+          <@fields.endDate />
+          <div class='divider' />
+          <@fields.endTime />
+        </div>
+      </FieldContainer>
+      <div class='column' />
+    </div>
+    <div class='row'>
+      <FieldContainer @label='Event type' @tag='label' class='field column'>
+        <BoxelDropdown @contentClass='context-menu'>
```

Comment:
can you use this component instead 
https://boxel-ui.stack.cards/?s=Components&ss=%3CSelect%3E